lib/cc/services/github_issues.rb

```diff
@@ -55,6 +55,10 @@ def receive_issue
 
 private
 
+  def base_url
+    ENV["CODECLIMATE_GITHUB_API_URL"] || DEFAULT_BASE_URL
```

Comment:
Hmm, I guess I'm confused why you think this code wouldn't be able to call `CodeClimate.config`, it's all in the same Ruby process, right?